erica/application/JobService/job.py

```diff
@@ -49,6 +50,7 @@ def perform_job(request_id: UUID, repository: base_repository_interface, service
             entity.result = response
             entity.status = Status.success
             repository.update(entity.id, entity)
+            logger.info(f"Job ran successfully: type {job_type}")
```

Comment:
Like the logging, but don't we log a similar thing (minus the request_type) in l.67?

Reply:
Having a differentiation definitely is the smarter move^^ So, yes, let's delete l.67

Reply:
Good catch. Had another look and l. 67 also logs the job type because this is part of the entity toString method. It also logs the entity status. Therefore, we can close this PR 🙈